packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1864,6 +1864,10 @@ export default mixins(
 								},
 							]);
 							this.zoomToFit();
+							this.$telemetry.track('welcome node inserted', {
```

Comment:
can you double check with Niv if we should still fire `User inserted workflow note` if this event is fired?

Reply:
Ok, will check.

Reply:
I am pasting the Slack response from @nivb06:

> I'd rather this to be as a different event for two reason
> 1. It's not a user that is inserting the welcome node sticky
> 2. We want the number of welcome node inserted to serve as a denominator for all other action that the user is doing afterwards

Reply:
thanks for checking